Gemfile

```diff
@@ -3,7 +3,8 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
-  gem 'activesupport', '< 5',      require: false
+  gem 'activesupport', "#{RUBY_VERSION < '2.2' ? '<' : '>'} 5", require: false
```

Comment:
For sure. However, since this logic will only be around until the EOL of Ruby 2.1, I think the simplicity of this approach makes sense.